api/models.py

```diff
@@ -26,3 +26,18 @@ def __str__(self):
         return "%s - %s (%s)" % (self.lava_job_id, self.lava_server, self.requester)
 
 
+class Submission(models.Model):
+    definition = models.TextField()
+
+    # fields that should be copied to Pattern
+    lava_server = models.URLField()
+    build_job_name = models.CharField(max_length=1024)
```

Comment:
yes, but  I am just using the same name it has in the existing `Pattern` class so they match.